sequences/tests.py

```diff
@@ -295,3 +303,37 @@ def two(output):
         ]
 
         self.assertSequence(one, two, expected)
+
+    def test_max_value_commit(self):
```

Comment:
Since the two sequences are independent, I don't believe this test validates a behavior that isn't already tested elsewhere.